lib/MusicBrainz/Server/Controller/Root.pm

```diff
@@ -71,6 +71,9 @@ sub set_language : Path('set-language') Args(1)
     } else {
         # set the cookie to expire in a year
         $c->set_language_cookie($lang);
+        $c->flash->{message} =
+            l('Language set. If you find any problems with the translation, please {url|help us improve it}!',
+              {url => {href => 'https://www.transifex.com/musicbrainz/musicbrainz/dashboard/', target => '_blank'}});
```

Comment:
The page you linked requires login. It might be better to use this one: https://www.transifex.com/musicbrainz/musicbrainz/.